code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -705,6 +705,29 @@ static HookFunction hookFunction([]()
 			}
 		});
 
+        fx::ScriptEngine::RegisterNativeHandler("MUMBLE_REMOVE_VOICE_TARGET_CHANNEL", [](fx::ScriptContext& context)
+        {
+            auto id = context.GetArgument<int>(0);
+            auto channel = context.GetArgument<int>(1);
+
+            if (id >= 0 && id < 31)
+            {
+                auto targetChannel = fmt::sprintf("Game Channel %d", channel);
+                std::vector<VoiceTargetConfig::Target>& targets = vtConfigs[id].targets;
+                for (size_t i = targets.size(); i--;)
+                {
+                    VoiceTargetConfig::Target& target = targets[i];
+                    if (!target.channel.empty() && target.channel.compare(targetChannel) == 0)
```

Comment:
Why `.compare()`?

Reply:
There wasn't really a reason, though now that I look back it looks bad syntactically